bundle/src/main/java/com/adobe/acs/commons/workflow/process/impl/WorkflowDelegationStep.java

```diff
@@ -11,11 +11,14 @@
 import org.apache.jackrabbit.JcrConstants;
 import org.apache.sling.api.resource.Resource;
 import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ValueMap;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.adobe.acs.commons.util.ParameterUtil;
 import com.adobe.acs.commons.util.WorkflowHelper;
+import com.adobe.granite.confmgr.Conf;
```

Comment:
The Granite Conf APIs are deprecated in AEM 6.3 [1] in favor of the Sling CAConfig APIs.

It would be good to check if CAConfig is available in AEM 6.2 and move to that if it is, else were introducing new code to Commons that uses deprecated APIs with 6.4's release right around the corner (which means we may/could drop 6.2 support in the main line of ACS Commons when its released)

https://helpx.adobe.com/experience-manager/6-3/sites/developing/using/reference-materials/javadoc/com/adobe/granite/confmgr/ConfMgr.html